serialize/src/lib.rs

```diff
@@ -12,10 +12,13 @@ mod error;
 mod flags;
 mod impls;
 
+mod serde;
```

Comment:
Or should this be `serde_with`?

```suggestion
#[cfg(feature = "serde")]
mod serde;
```

Reply:
i wonder if it's a good idea to keep the marker types unconditionally available, because they have minor utility for type-driven serialozation mode customization? seems rarely useful or wanted though.

as a zero-cost annotation they could foreseeably be used in public APIs.

but i'm not sure it helps in practice

Reply:
Hm that's a good idea. Sure, we can keep them available.